teachertool/src/App.tsx

```diff
@@ -28,6 +34,10 @@ function App() {
                 const cfg = await downloadTargetConfigAsync();
                 dispatch(Actions.setTargetConfig(cfg || {}));
                 pxt.BrowserUtils.initTheme();
+
+                // Load criteria catalog
+                await loadCatalogAsync();
```

Comment:
@eanders-ms I wasn't totally sure how to plug into this initialization code. Is this suitable?

Reply:
Yep, looks good! If we end up with a lot of async tasks here we should try to parallelize some of them. Nothing to worry about yet.